main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/gorilla/mux"
+	"net/http"
+)
+
+type ScreenShot struct {
+	Width    int64  `json:"width"`
+	Height   int64  `json:"height"`
+	Filename string `json:"filename"`
+}
+
+type Ticket struct {
+	ID         int64      `json:"id"`
+	Name       string     `json:"name"`
+	URL        string     `json:"url"`
+	Screenshot ScreenShot `json:"screenshot"`
+}
+
+func main() {
+	handler := mux.NewRouter()                 // create router for handling api endpoints
+	handler.HandleFunc("/api/hello", SayHello) // hello world endpoint
+	handler.HandleFunc("/api/createClient", CreateClient)
```

Comment:
Should we use something like `/api/create_client` or `/api/create-client` if the user will see this in the URL? Because the camel case in the URL looks really wried to me.

Reply:
hmm good call, I just googled this and the good people of stackoverflow are saying hyphens are the way to go. I can update this pull request tomorrow

![Screen Shot 2019-10-07 at 8 28 15 PM](https://user-images.githubusercontent.com/27692412/66360525-179fb780-e941-11e9-84a5-d51762325160.png)